packages/client/bin/cli.ts

```diff
@@ -191,17 +195,21 @@ async function run() {
   // configure common based on args given
   let common: Common = {} as Common
   // Use custom chain parameters file if specified
-  if (args.params) {
+  if (args.customGenesis) {
     try {
-      const params = JSON.parse(fs.readFileSync(args.params, 'utf-8'))
-      common = new Common({ chain: params })
+      const genesisParams = JSON.parse(fs.readFileSync(args.customGenesis, 'utf-8'))
+      const genesisState = args.readFileSync(args.customGenesisState) ?? {}
+      common = new Common({
+        chain: args.customGenesis.split('.')[0],
```

Comment:
Maybe I am missing something, what is this `.split('.')[0]` doing? Tought this should very much take in our file format 1:1?

Reply:
You're right.  I corrected this in a follow-on commit where the chain name is pulled from the `name` property of the `Common` object.